goal_src/jak2/engine/game/task/task-control.gc

```diff
@@ -704,19 +704,25 @@
 (defun get-active-mission-description ()
   (let* ((nodes (-> *game-info* sub-task-list))
          (level (level-get-target-inside *level*))
-         (task-level-idx (if level (-> level info task-level) 0)))
+         (task-level-idx (if level (-> level info task-level) 0))
+         (side-mission-idx (the int (game-task-node city-burning-bush-ring-1-introduction)))
+         (side-mission-end (the int (game-task-node stadium-burning-bush-race-class1-r-resolution)))
+         )
+    ;; check if a side mission is active
+    (while (< side-mission-idx side-mission-end)
```

Comment:
you want `<=`, to include `stadium-burning-bush-race-class1-r-resolution`.

Reply:
I completely refactored the function, this should be fixed now.